src/main/webapp/chatroom.html

```diff
@@ -26,8 +26,37 @@
   </head>
   <body>
 
-    <div id="map" class="expanded"></div>
+    <div id="mapOverlay" class="expanded">
```

Comment:
Nice! This is much better than using createElement to build it out!

Reply:
I can't believe I didn't think of this before, it was really satisfying to delete all the button-building functions in ChapMap 😆